docs/docs/reference/dropped/weak-conformance-spec.md

```diff
@@ -36,11 +36,11 @@ __Examples:__
 
     inline val b = 33
     def f(): Int = b + 1
-    List(b, 33, 5.5)      : List[Double] // b is an inline val
-    List(f(), 33, 5.5)    : List[AnyVal] // f() is not a constant
-    List(5, 11L)          : List[Long]
-    List(5, 11L, 5.5)     : List[AnyVal] // Long and Double found
-    List(1.0f, 2)         : List[Float]
-    List(1.0f, 1234567890): List[AnyVal] // loss of precision
-    List(b, 33, 'a')      : List[AnyVal] // Char is not a numeric
-    List(5.toByte, 11)    : List[Byte]
+    Array(b, 33, 5.5)      : Array[Double] // b is an inline val
+    Array(f(), 33, 5.5)    : Array[AnyVal] // f() is not a constant
+    Array(5, 11L)          : Array[Long]
+    Array(5, 11L, 5.5)     : Array[AnyVal] // Long and Double found
+    Array(1.0f, 2)         : Array[Float]
+    Array(1.0f, 1234567890): Array[AnyVal] // loss of precision
+    Array(b, 33, 'a')      : Array[Char]
```

Comment:
I thought we agreed that a Char literal should not be converted to another numeric type, same as a Float or a Double is not converted. 

But this case is different: It's an Int constant that is converted to a Char type. Same as it could be converted to a Short or Byte type. I agree that Char feels strange, but the fact is it is a member of Java's numeric hierarchy, so I believe it's better to be consistent.

We do allow:
```
scala> val c: Char = 65
c: Char = A
```
So it's inconsistent to not also allow this in harmonization.